### PR TITLE
[script][tinker] bolts in assembly/chapter check

### PR DESCRIPTION
**Pull request by Raykyn55**
chapter wasn't properly checking against #, and missing the flag language for bolt bits.
Also added specific tools grabbing, and eliminated period in join.

### CODE REVIEW

**Comment by MahtraDR** [comment]
Resolves https://github.com/rpherbig/dr-scripts/issues/6385

**Comment by Raykyn55** [comment]
Should be sorted. Arrows with shape looks to be working fine.

**Comment by MahtraDR** [comment]
The version in this PR gets stuck on stain ```

2023-02-13 14:46:44 NZDT: --- Lich: tinker active.
2023-02-13 14:46:44 NZDT: >;tinker 7 "drake-fang bolts" "shafts" "bolts"
2023-02-13 14:46:44 NZDT: --- Lich: buff active.
2023-02-13 14:46:44 NZDT: <pushBold/>No waggle set found for name: tinker<popBold/>
2023-02-13 14:46:44 NZDT: --- Lich: buff has exited.
2023-02-13 14:46:46 NZDT: [tinker]>get my stain from my duffel bag
2023-02-13 14:46:47 NZDT: What were you referring to?
2023-02-13 14:46:47 NZDT: [tinker]>order 
2023-02-13 14:46:47 NZDT: Order what?
``` And all goes to custard after this.

**Comment by MahtraDR** [comment]
`@info` isn't defined in the current version of the script in the repo. This can't work...

File: tinker.lic [hunk 1]
```diff
@@ -50,13 +50,10 @@ class Tinker
...
     args = parse_args(arg_definitions)
-
-    @finish = args.finish
     @chapter = args.chapter
     @material = args.material
     @mechanism = args.mechanism
-    @home_tool = 'shaper'
-    @chapter = args.chapter ? args.chapter : 8
+    @chapter = args.chapter ? args.chapter.to_i : 8
     @noun = args.noun
```
Comment (MahtraDR) [inline tinker.lic:57]:
```suggestion
    @noun = args.noun
    @info = get_data('crafting')['shaping'][@hometown]
```

Comment (MahtraDR) [reply tinker.lic:57]: With `@info` defined as above, I managed to successfuly make drake-fang bolts.

**Comment by Raykyn55** [comment]
Thanks @MahtraDR

**Pull request merged**